app/src/main/java/org/dhis2/usescases/teiDashboard/ui/mapper/TeiDashboardCardMapper.kt

```diff
@@ -265,4 +265,5 @@ class TeiDashboardCardMapper(
         this.filter { it.first.valueType() != ValueType.IMAGE }
             .filter { it.first.valueType() != ValueType.COORDINATE }
             .filter { it.first.valueType() != ValueType.FILE_RESOURCE }
+            .filter { it.second.value()?.isNotEmpty() == true }
```

Comment:
Is this fix related with the issue?

Reply:
It is not with the issue itself, but Marcos mentioned it in the troubleshooting channel that we have apparently lost this functionality and as it's an easy fix I thought I would include it